java/client/src/main/java/glide/api/commands/StringBaseCommands.java

```diff
@@ -156,6 +156,25 @@ public interface StringBaseCommands {
      */
     CompletableFuture<String> mset(Map<String, String> keyValueMap);
 
+    /**
+     * Sets multiple keys to multiple values in a single operation. Performs not operation at all even
```

Comment:
Suggest re-wording to make it clear that it only sets keys if the key does not exist.  
```
* Sets multiple keys to values if the key does not exist. The operation is atomic, and if one or more keys already exist, the entire operation fails. 
```